disttask/framework/dispatcher/dispatcher.go

```diff
@@ -74,211 +63,233 @@ type dispatcher struct {
 	ctx     context.Context
 	taskMgr *storage.TaskManager
 	task    *proto.Task
+	logCtx  context.Context
 }
 
 // MockOwnerChange mock owner change in tests.
 var MockOwnerChange func()
 
 func newDispatcher(ctx context.Context, taskMgr *storage.TaskManager, task *proto.Task) *dispatcher {
+	logPrefix := fmt.Sprintf("task_id: %d, task_type: %s", task.ID, task.Type)
 	return &dispatcher{
 		ctx,
 		taskMgr,
 		task,
+		logutil.WithKeyValue(context.Background(), "dispatcher", logPrefix),
 	}
 }
 
-// ExecuteTask start to schedule a task
-func (d *dispatcher) ExecuteTask() {
-	logutil.BgLogger().Info("execute one task", zap.Int64("task ID", d.task.ID),
+// ExecuteTask start to schedule a task.
+func (d *dispatcher) executeTask() {
```

Comment:
Maybe remove this function since the body is another function?

Reply:
> Maybe remove this function since the body is another function?

History task management will be added after scheduleTask() later on. So I add a executeTask function.